OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -120,6 +124,7 @@ public void setUserVisibleHint(boolean isVisibleToUser) {
     public void onResume() {
         super.onResume();
         authWebView.onResume();
+        swipeContainer.setEnabled(false);
```

Comment:
When are we are disabling the swipe container in the `onResume` callback not in `onCreate` callback.